### PR TITLE
Get extra metadata from the content service

### PR DESCRIPTION
**Pull request by samgiles**
Get the extra metadata from the content service and embellish the metadata response with it.

### CODE REVIEW

**Comment by samgiles** [comment]
@wilsonpage  r?

Closes #60

File: lib/routes/api/metadata/processor.js [hunk 1]
```diff
@@ -56,21 +61,30 @@ module.exports = function(url, options) {
...
-        const twitterUsername = result.foundParams.magnet_twitter_username;
-        if (twitterUsername) {
-          metadata.twitterUsername = twitterUsername.replace('@', '');
-        }
-
-        return metadata;
+        return contentServiceRequest.then((res) => {
+          return Object.assign(metadata, res.extra_metadata || {});
+        })
+        .catch((err) => {
```
Comment (wilsonpage) [inline lib/routes/api/metadata/processor.js:67]: Can you add a little comment about when/how a URL could error and why it doesn't matter.

**Review by wilsonpage — COMMENTED** [review]
r+

File: lib/processors/contentservice.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const request = require('superagent');
```
Comment (wilsonpage) [inline lib/processors/contentservice.js]: Filename should be `-` separated:

`content-service.js`

**Pull request merged**